.github/workflows/build-deploy-pudl.yml

```diff
@@ -22,17 +21,15 @@ jobs:
       contents: write
       id-token: write
     steps:
-      - name: Use pudl-deployment-dev vm and dev branch if running on a schedule
+      - name: Use pudl-deployment-dev vm if running on a schedule
         if: ${{ (github.event_name == 'schedule') }}
         run: |
           echo "This action was triggered by a schedule."
           echo "GCE_INSTANCE=pudl-deployment-dev" >> $GITHUB_ENV
```

Comment:
Hopefully we'll be on Batch shortly and the archaic naming of this VM won't matter.

Reply:
I also think, at some point, we just trigger the whole `build-deploy` only on tag push and workflow dispatch, and just make the nightly tag on a schedule. Which will mean we don't have to do all these conditionals on the event name.

Reply:
I've been thinking it might be nice to completely separate the build and deploy processes.

Like the build runs the ETL and tests / validations and fills up a directory in `gs://builds.catalyst.coop` as it does now, and then a totally separate script could be used to update the `nightly` branch, and deploy the outputs from a selected `BUILD_ID` to any combination of:

* Datasette on Fly.io
* AWS Open Data Registry S3 buckets
* Our GCS buckets that mirror AWS
* ...